engine/drones/Eliane.scd

```diff
@@ -0,0 +1,54 @@
+// @sixolet
+// Eliane
+// Feedback, slow beatings, highs and lows
+
+// (seven sines phase-modulating each other in a crosslinked ring)
+
+{ 
+  arg hz=220,amp=1.0;
+	var base = 2.pow(1/12.0);
+	// All the notes in a natural minor scale, but by fifths (with one diminished).
+	// root, fifth, second + 8v, minor sixth +8v, minor third +2 8v, minor seventh +2 8v, fourth +3 8v 
+	var intervals = base.pow([0, 7, 14, 20, 27, 34, 41]);
+	var busses = LocalIn.ar(14);
+	var nice = LFNoise2.kr(200.reciprocal);
+	var bigsound;
+	var sines = 7.collect { |x|
+		var next = (x+1)%7;
+		var nextayim = (x+2)%7;
+		var modulator1 = [busses[2*next], busses[2*next+1]];
```

Comment:
Genuine question - is this
`[busses[2*next], busses[2*next+1]]`
different from this?
`busses[2*next + [0,1]]`
Actually now that I typed it out, it's kinda ugly, isn't it.

Reply:
I have sorted this using another reshape